CHANGELOG.md

```diff
@@ -5,34 +5,39 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#63)
 
 New features:
+- Replaced monomorphic proxies with `Type.Proxy.Proxy` and polymorphic variables and made `If` and `TypeEquals` polykinded (#64)
 
```

Comment:
#63 also added `APPLY` and `FLIP`. Perhaps this is how it could be worded?
```
- Added `APPLY`/`$` and `FLIP` type aliases (#63)
     - The type `Foo (Bar Baz)` is the same as `Foo $ Bar Baz`
     - The type `Either a b` is the same as `FLIP Either b a`
```